rules-tests/CodingStyle/NodeAnalyzer/UseImportNameMatcher/Fixture/UseClass.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Rector\Tests\CodingStyle\NodeAnalyzer\UseImportNameMatcher\Fixture;
+
+
+use Rector\Tests\CodingStyle\NodeAnalyzer\UseImportNameMatcher\Fixture\FixtureAnnotation;
+use Rector\Tests\CodingStyle\NodeAnalyzer\UseImportNameMatcher\Fixture\FixtureAnnotation as Test;
+
+class UseClass
+{
+    /**
+     * @var float
+     *
+     * @FixtureAnnotation\CustomAnnotation()
```

Comment:
it seems need a test with `\` prefix for fqcn ensure for:

```
     * @\FixtureAnnotation\CustomAnnotation()
```

ensure that different namespace for FQCN?

Reply:
Not sure I understand, is annotation starting with `@\` exists ?

Reply:
Yes, annotation work with `@\` usage as FQCN